core-bundle/contao/pages/PageRegular.php

```diff
@@ -621,7 +621,15 @@ protected function createFooterScripts($objPage, $objLayout)
 		// Add the custom JavaScript
 		if ($objLayout->script)
 		{
-			$strScripts .= "\n" . trim($objLayout->script) . "\n";
+			$customScript = trim($objLayout->script);
+
+			// Add nonces to the <script> tags since we consider this safe user input.
```

Comment:
```suggestion
			// Add nonces to the <script> tags since we consider this safe user input.
			// Do NOT copy the str_replace() into your own code unless you know what you're doing - it will defeat the purpose of CSP.
```